tests/app/notifications/test_process_notification.py

```diff
@@ -265,17 +268,24 @@ def test_send_notification_to_queue(
     requested_queue,
     notification_type,
     key_type,
+    reply_to_text,
     expected_queue,
     expected_task,
     mocker,
 ):
-    mocked = mocker.patch('app.celery.{}.apply_async'.format(expected_task))
-    Notification = namedtuple('Notification', ['id', 'key_type', 'notification_type', 'created_at'])
+    if '.' not in expected_task:
+        expected_task = f'provider_tasks.{expected_task}'
```

Comment:
This dance is used to avoid long lines when setting up the test data, flake8 isn't happy otherwise